lib/lint.js

```diff
@@ -0,0 +1,325 @@
+'use babel';
+
+function declarationSupportedValues(declarationName) {
+	switch (declarationName) {
+		case 'indent_style': return ['tab', 'space'];
+		case 'end_of_line': return ['lf', 'cr', 'crlf'];
+		case 'charset': return ['latin1', 'utf-8', 'utf-8-bom', 'utf-16be', 'utf-16le'];
```

Comment:
This reminds me; the character sets supported by EditorConfig files aren't set in stone, and implementations are free to support other encodings too. The list above is only the [barest minimum](https://github.com/editorconfig/editorconfig/wiki/Character-Set-Support#supported-character-sets) that EditorConfig plugins are expected to support.

I've yet to update the plugin for this, but in the meantime, it might be best to avoid enforcing a fixed set of `charset` values.

Reply:
Done, it no more validate for a specific list of values for charset… it now only check that a value is provided

Reply:
Excellent, thanks. I'll try to look at this tomorrow. 👍